elementary/exceptions/exceptions.py

```diff
@@ -82,6 +82,17 @@ def extract_detailed_dbt_command_args(command_args):
         except Exception as ex:
             logger.error(f"Failed to extract detailed dbt command args, error: {ex}")
 
+    def get_exception_message(self) -> Optional[str]:
+        error_output = self.proc_err.output.decode()
+        for line in reversed(error_output.strip().splitlines()):
```

Comment:
Note that in other versions of dbt-core the `data` was renamed to `info`.
You can see an example of how we parse it [here](https://github.com/elementary-data/elementary/blob/4135a4530e481a49b9905e39a284a9adb39c72ec/elementary/clients/dbt/dbt_runner.py#L16) and [here](https://github.com/elementary-data/elementary/blob/4135a4530e481a49b9905e39a284a9adb39c72ec/elementary/clients/dbt/dbt_runner.py#L262).

I'd recommend we export this logic and share it between the two.